.travis.yml

```diff
@@ -2,7 +2,7 @@ env:
     global:
         - REPO_DIR=scipy
         # Also see DAILY_COMMIT below
-        - BUILD_COMMIT=ab1c0907f
+        - BUILD_COMMIT=68eb089
```

Comment:
A few thoughts here:

1) Why not open the PR against `master`? Unless there's a special reason to do otherwise, we'd normally do that and backport from there if we really need a backport.
2) If you really want to build your specific feature branch, you can try setting `BUILD_COMMIT` to the name of your feature branch (if you open a PR against `master` you'll notice that `BUILD_COMMIT` points at `master`.
3) I don't see anything special about your feature branch at the moment in terms of things the wheels repo would care about--I assume your plan is to make meaningful changes in that feature branch in response to any issues that crop up here in the wheels repo.

Reply:
@tylerjereddy Thanks, I think with your feedback I understand what's going on a little bit better.
1. Yes, I can open a new PR against master
2. If I open a PR against master, does `DAILY_COMMIT` need to change to the branch name?  or do I just need to make `BUILD_COMMIT=mckib2:boost`?
3. I was pointing to the wrong branch --  I will need to point to the actual feature branch that we need testing for (https://github.com/scipy/scipy/pull/13328) and modify the scripts here to use gcc5 and then mirror those changes in mckib2:gcc48to52 if any are needed

I will open a new PR tomorrow, thanks!